libkeepass/kdb4.py

```diff
@@ -90,7 +90,7 @@ def write_to(self, stream):
         
         :arg stream: A writeable file-like object or IO buffer.
         """
-        if not (isinstance(stream, io.IOBase) or isinstance(stream, file)):
+        if not (isinstance(stream, io.IOBase)) or (not IS_PYTHON_3 and not isinstance(stream, file)):
```

Comment:
We'll probably drop Python 2 support at some point. But this here is not place nor the time to discuss this.